.github/Issue_Template/Doc_report.yml

```diff
@@ -0,0 +1,18 @@
+name: Documentation request
+description: Change regarding improving the docs to be more accessible
+title: '[docs]'
+labels: ['documentation']
+body:
+  - type: textarea
+    attributes:
+      label: Category of documentation update
+      description: |
+        What category does this change fall under.For example Typo error, New category addition, Rephrasing the sentences, fixing broken links etc 
```

Comment:
Please correct the grammar like a space after the `.`, `:` after `For example` etc